Vic2ToHoI4/Data_Files/configurables/IdeologyMappings.txt

```diff
@@ -0,0 +1,48 @@
+mapping = { ruling_ideology = absolutist vic2_ideology = reactionary supported_ideology = absolutist }
```

Comment:
Instructions. 
You're mapping country and party ideologies into hoi4 ideology? Which is which?

Does 
```
mapping = { ruling_ideology = absolutist vic2_ideology = social_liberal supported_ideology = democratic }
```
what I assume is social liberals in charge of an absolutist country country map to democratic? 

If so, replace ruling ideology with party_ideology and supported_ideology with hoi4_ideology.

Reply:
I've added instructions, so hopefully more clear. I think the names still stand, but let me know if you see room for improvement.